packages/design-system/src/icons/align_text_center.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg" data-type="center">
```

Comment:
This is kind of a cheat, but it works!

I saw no better way to test in code, that the actual icon displayed was the correct one. The only alternative was to hardcode the SVG paths, but that definitely seemed like a worse option.

More semantic alternatives include putting a title on the whole SVG (`<svg ...><title>Align text center</title><path .../></svg>`) or on the individual elements (`<svg ...><path ...><title>Align text center</title></path></svg>`), but we strip this kind of metadata from our SVG's in svgo, so that's no good. The `data-` property however survives.

Reply:
Changed in a145fc96d1.